packages/mdc-fab/README.md

```diff
@@ -59,6 +59,16 @@ We recommend you load [Material Icons](https://material.io/icons/) from Google F
 
 > _NOTE:_ The floating action button icon can be used with a `span`, `i`, `img`, or `svg` element.
 
+### Styles
+
+```scss
+@import "@material/fab/mdc-fab";
+
+// Include necessary mixin invocations here
```

Comment:
Remove this, this is guidance in the readme template. There are no required invocations for FAB.